lib/video_info.rb

```diff
@@ -41,6 +41,19 @@ def ==(other)
     url == other.url && video_id == other.video_id
   end
 
+  @@provider_api_keys = {}
+
+  def self.provider_api_keys
+    @@provider_api_keys
+  end
+
+  def self.provider_api_keys=(api_keys)
+    api_keys.keys.each do |key|
+      raise ArgumentError, "Key must be a symbol!" unless key.is_a?(Symbol)
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.